lib/index.js

```diff
@@ -224,7 +237,7 @@ internals.models = (serverFrom, realmPath) => {
             return collector.models;
         }
 
-        const knexGroup = Hoek.reach(this, `${realmPath}.plugins.schwifty.knexGroup`);
+        const knexGroup = getKnexGroup(this);
```

Comment:
More of a question than a complaint - why not just call `Toys.reacher` from here?  I noticed the same pattern in the toys docs - my inclination would be to avoid the `const` in cases where it was only being used once, but I don't really have a good argument either way.